app/views/volunteers/show.html.slim

```diff
@@ -158,8 +168,10 @@ h3= t('volunteer_availability')
       h3= t('assignments')
   = render 'assignments/volunteer_index', assignments: @volunteer.assignments
 
-- if current_user.volunteer?
-  = button_link t('edit_profile'), edit_volunteer_path(@volunteer)
-- else
-  = form_navigation_btn :edit
-  = form_navigation_btn :back
+nav.navbar.section-navigation
+  ul.list-inline
+    - if current_user.volunteer?
```

Comment:
Maybe reuse a policy here?

Reply:
The policy's will only give a true or false in case the user is a superadmin or if the user is volunteer and the record is associated to it. I'd have to create a new policy rule for this, and as this condition is only for what text is beeing displayed for the same link. So actually not a policy matter at all.